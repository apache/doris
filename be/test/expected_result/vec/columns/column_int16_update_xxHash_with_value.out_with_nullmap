column: Int16 with hashes: 17235753159725647188 with ptr: 84