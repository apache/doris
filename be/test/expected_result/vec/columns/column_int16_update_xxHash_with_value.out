column: Int16 with hashes: 12110887499945165499 with ptr: 84